hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/invalidation/BatchInvalidator.java

```diff
@@ -133,10 +133,11 @@ private void createAndSendInvalidations(String mapName, InvalidationQueue invali
     }
 
     private void sendInvalidations(String mapName, List<Invalidation> invalidations) {
+        Invalidation invalidation = new BatchNearCacheInvalidation(mapName, invalidations);
```

Comment:
Then I would say: good work and :+1: 

Can you add a line of comment explaining the assumption? For future generations..